src/sdg/accounts/migrations/0015_update_ontvangt_email.py

```diff
@@ -0,0 +1,26 @@
+# Generated by Django 3.2.16 on 2023-04-05 13:17
+
+from django.db import migrations
+from django.db.models import Q
+
+
+def update_ontvangt_mail_boolean_for_beheerder_and_redacteur(apps, schema_editor):
+    roles = apps.get_model("accounts", "Role")
+    roles = roles.objects.filter(Q(is_beheerder=True) | Q(is_redacteur=True))
+    for role in roles:
+        role.ontvangt_mail = True
+        role.save()
```

Comment:
Convention is to use the modelclass name for this var. Furthermore, this can be a single db-query instead of a Python loop does a db-query for everyone.

```suggestion
    Role = apps.get_model("accounts", "Role")
    Role.objects.filter(Q(is_beheerder=True) | Q(is_redacteur=True)).update(ontvangt_mail=True)
```